console/src/components/Services/Settings/utils.ts

```diff
@@ -5,18 +5,41 @@ export const permissionTypes = [
   'delete_permission',
 ];
 
-export const getTableNameFromDef = def => {
-  if (def.constructor.name === 'Object') {
+export type TableDef = {
```

Comment:
It looks like something used only in this particular use case (for `getTableNameFromDef`). And as in some other places, we have table definition represented as `{ table_name: string }` or `{ tableName: string }` or `{ name: string, other_attribute: Type }`etc, I wouldn't export this type. (At least not until we have all of these types unified).

What's more, I would just inline this type in the function definition:
```ts
export const getTableNameFromDef = (def: { name: string } | string) => {
```